### DISCUSSION TITLE
Support MED AFRICA LOGISTICS As Shipping Carrier

### DISCUSSION THREAD

**Discussion by Youssef-ELBADRI** [etsy/open-api#999] [Q&A]
Hi, 

@etsyachristensen Please we need your help !
We would like to request to add "Med Africa Logistics" to the list of carriers that can be trackable on Etsy.

Med Africa Logistics is a shipping carrier, we are the representative of Skynet worldwide UK in Morocco.

As mentioned in some discussions. we need to provide the informations below :
1- Example URLs for several shipments. We need to see what the url structure looks like such that we can direct users to a common url with a tracking code. For example: https://track.somecarrier.com/?tracking=xxxxxxxxx

https://www.aftership.com/track/medafrica/013090004461
https://www.aftership.com/track/medafrica/013090004401
https://www.aftership.com/track/medafrica/013090004452

1- Example of some tracking codes (they can be fake, but must match the typical formatting of a tracking code for the carrier).

**Tracking From Skynetworldwide UK directly :**
- Tracking Download Endpoint :
https://ws05.ffdx.net/ffdx_ws/[version no]/service_ffdx.asmx/WSDataTransfer

- Download Event For Single Shipment
![image](https://github.com/etsy/open-api/assets/133863885/7d34a8f6-9486-496b-b680-a5782b12b6b0)

                           
**Tracking From Postshipping :**
https://doc.postshipping.com/docs/developers/tracks/

@etsyachristensen Please if we need to setup an integration API. we are ready for that (Rest or SOAP API).

Thanks in advance
Youssef ELBADRI

**Comment by Gareth064** [comment]
Hi @Youssef-ELBADRI 

etsyachristensen no longer works for Etsy.

I would send your request to developer@etsy.com instead as they can help you there.

**Reply by Youssef-ELBADRI** [reply]
Hi @Gareth064 

Thank you so much for your reply.

We need your help on this request, would you please help us?

**Reply by Gareth064** [reply]
Sorry, I don't work for Etsy. No-one on here, from the community, can help with this request. You will need to email the Etsy developers for help on this one 👍🏻

**Reply by Youssef-ELBADRI** [reply]
Ok. Thank you for your time 👍
